packages/ts-types/README.md

```diff
@@ -95,17 +123,13 @@ that supports TypeScript definitions based languages services and start coding.
 
 #### sap.ui.define Imports
 
-For UI5 source code using the newer `sap.ui.define` syntax, an additional JSDocs parameter is needed.
-This parameter "helps" the TypeScript compiler understand the UI5 import syntax and "link" to the correct types signature.
+For UI5 source code using the newer `sap.ui.define` syntax, an additional type declaration is needed.
+This "helps" the TypeScript compiler understand the UI5 import syntax and "link" to the correct types signature.
 
 ```javascript
 sap.ui.define(
   ["sap/ui/core/TooltipBase"], // 1. UI5 runtime import
-  /**
-   *        // 2. Linking TypeScript global signatures (below)
-   * @param {typeof sap.ui.core.TooltipBase} TooltipBase
-   */
-  function(TooltipBase) {
+  function(TooltipBase: typeof sap.ui.core.TooltipBase) { // 2. Type declaration
```

Comment:
That is the Typescript syntax example, I also want to document the JavaScript scenario, I even prefer to prioritize the JavaScript scenario.

Reply:
Ah, I see. I'm focused on typescript. My fault. 
Me and a few other guys are eager to develop UI5 applications with typescript.

Reply:
maybe the main README.md should reference two **separate** documents,
- One for JavaScript.
- One for TypeScript.

And have the demos align to the instructions in those guides?
- https://github.com/SAP/ui5-typescript/tree/master/demos